libcob/common.c

```diff
@@ -1519,10 +1522,17 @@ cob_set_signal (void)
 {
 #if	defined (HAVE_SIGNAL_H)
 	int k;
+	char *signal_regime;
```

Comment:
please replace by

```c
	char *s = getenv ("COB_SIGNAL_REGIME");
	char signal_regime = s ? *s : '0';
```

this way you can use `signal_regime` directly in the code below and reduce duplicated code